components/brave_rewards/core/contribution/contribution.cc

```diff
@@ -852,9 +842,8 @@ void Contribution::Retry(mojom::ContributionInfoPtr contribution,
 
   switch (contribution->processor) {
     case mojom::ContributionProcessor::BRAVE_TOKENS: {
-      RetryUnblindedContribution({mojom::CredsBatchType::PROMOTION},
-                                 std::move(result_callback),
-                                 std::move(contribution));
+      // TODO(zenparsing): Remove this enum value?
```

Comment:
`ContributionProcessor::BRAVE_TOKENS`? I think so.

Reply:
Thanks for pointing this out. I created a follow-up issue to remove these and other remaining vBAT references:

https://github.com/brave/brave-browser/issues/36936